onebusaway-android/src/main/java/org/onebusaway/android/report/ui/Open311ProblemFragment.java

```diff
@@ -0,0 +1,1070 @@
+            mIssueImage.setImageBitmap((Bitmap) bundle.getParcelable(IMAGE));
+        }
+
+        mInfoLayout = (LinearLayout) findViewById(R.id.ri_info_layout);
+
+        Button addImageButton = (Button) findViewById(R.id.ri_attach_image);
+
+        final PopupMenu popupMenu = new PopupMenu(getActivity(), addImageButton);
+        popupMenu.inflate(R.menu.report_issue_add_image);
+        addImageButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                popupMenu.show();
+            }
+        });
+
+        popupMenu.setOnMenuItemClickListener(
+                new PopupMenu.OnMenuItemClickListener() {
+                    @Override
+                    public boolean onMenuItemClick(MenuItem item) {
+                        switch (item.getItemId()) {
+                            case R.id.ri_button_camera:
+                                openCamera();
+                                break;
+                            case R.id.ri_button_gallery:
+                                openGallery();
+                                break;
+                            default:
+                                break;
+                        }
+                        return true;
+                    }
+                });
+
+        mAnonymousReportingCheckBox = (CheckBox) findViewById(R.id.rici_anonymous_checkbox);
+        mAnonymousReportingCheckBox.setOnCheckedChangeListener(new CompoundButton.OnCheckedChangeListener() {
+            @Override
+            public void onCheckedChanged(CompoundButton compoundButton, boolean isChecked) {
+                disableEnableContactInfoViews(isChecked);
+            }
+        });
+
+        // Setup contact info views
+        mContactNameView = ((EditText) findViewById(R.id.rici_name_editText));
+        mContactLastNameView = ((EditText) findViewById(R.id.rici_lastname_editText));
+        mContactEmailView = ((EditText) findViewById(R.id.rici_email_editText));
+        mContactPhoneView = ((EditText) findViewById(R.id.rici_phone_editText));
+    }
+
+    private void setupIconColors() {
+        ((ImageView) findViewById(R.id.ri_ic_app_feedback)).setColorFilter(
+                getResources().getColor(R.color.material_gray));
+        ((ImageView) findViewById(R.id.ri_ic_image_picker)).setColorFilter(
+                getResources().getColor(R.color.material_gray));
+        ((ImageView) findViewById(R.id.ri_ic_username)).setColorFilter(
+                getResources().getColor(R.color.material_gray));
+        ((ImageView) findViewById(R.id.ri_ic_customer_service_email)).setColorFilter(
+                getResources().getColor(R.color.material_gray));
+        ((ImageView) findViewById(R.id.ri_ic_customer_service_phone)).setColorFilter(
+                getResources().getColor(R.color.material_gray));
+        ((ImageView) findViewById(R.id.ri_ic_anonymous)).setColorFilter(
```

Comment:
I think the anonymous icon wasn't an official Google material design icon, but was from https://materialdesignicons.com/?  I think https://materialdesignicons.com/?  If so we need to add Christopher Schreiner to the About screen under the icon credits section.  If any of the other icons also aren't from Google, we need to do the same for them too.